assets/pages/bundles-landing/bundlesLanding.scss

```diff
@@ -526,7 +525,7 @@
       color: gu-colour(neutral-1);
 
       h3 {
-        color: gu-colour(guardian-brand);
+        color: gu-colour(sport-garnett-main-1);
```

Comment:
Do we want to keep the garnett name? I would change it to `sport-main-1`

Reply:
It's necessary for two reasons:

1. We're still using the old palette colours alongside, and they all use these names too.
2. This is what they're called in the main palette on dotcom.